### DISCUSSION TITLE
Draggable list - rows of varying heights

### DISCUSSION THREAD

**Discussion by david-buck** [pmndrs/use-gesture#582] [Q&A] [answered]
Hi There

I've been trying to add drag and drop reordering to my app basing it on the implementation in the Draggable List demo (currently broken?) https://codesandbox.io/s/github/pmndrs/use-gesture/tree/main/demo/src/sandboxes/draggable-list

The issue is that my rows are of varying heights, so I need to be able to reorder based on the actual heights of the rows, not just a constant value. 

Has anyone had any luck with that? I'm yet to find an example using any library that does this.

Thanks

**Comment by dbismut** [comment, marked as answer]
See discussion here: https://github.com/pmndrs/use-gesture/pull/560

Issues: https://github.com/pmndrs/use-gesture/issues/556, https://github.com/pmndrs/use-gesture/issues/565, https://github.com/pmndrs/use-gesture/issues/572, https://github.com/pmndrs/use-gesture/issues/579

This is on CSB side. In the meantime, you can download the examples locally.
